.changeset/tall-lies-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
+---
+
+update translations of Korean UI strings
```

Comment:
Suggesting a small tweak to specify that we’re improving as well as adding the missing translations in this patch:

```suggestion
Updates Korean langage support with improvements and missing translations
```

Reply:
Thanks for good suggestion.